src/pages/AddPersonalBankAccountPage.js

```diff
@@ -24,21 +25,23 @@ const defaultProps = {
 
 const AddPersonalBankAccountPage = props => (
     <ScreenWrapper>
-        <HeaderWithCloseButton
-            title={props.translate('bankAccount.addBankAccount')}
-            onCloseButtonPress={Navigation.dismissModal}
-            shouldShowBackButton
-            onBackButtonPress={() => Navigation.goBack()}
-        />
-        <AddPlaidBankAccount
-            onSubmit={({account, password, plaidLinkToken}) => {
-                BankAccounts.addPersonalBankAccount(account, password, plaidLinkToken);
-            }}
-            onExitPlaid={Navigation.dismissModal}
-            receivedRedirectURI={getPlaidOAuthReceivedRedirectURI()}
-            plaidLinkOAuthToken={props.plaidLinkToken}
-            isPasswordRequired
-        />
+        <KeyboardAvoidingView>
+            <HeaderWithCloseButton
+                title={props.translate('bankAccount.addBankAccount')}
+                onCloseButtonPress={Navigation.dismissModal}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.goBack()}
+            />
```

Comment:
This should be outside the `KeyboardAvoidingView`.

Reply:
Why would that be?

Reply:
Headers are fixed all the time. we don't want to adjust them based on the keyboard. It makes the code more clear to just wrap the main content and the whole app follows that pattern. :cat:

Reply:
The headers are fixed, but every component I've looked at in the app follows the pattern of having `KeyboardAvoidingView` outside of the `Header`

Reply:
Oh, man. I am just confused. You are correct. I was thinking we put the header outside but that was Scrollview. Sorry for the confusion.

Reply:
All good! Appreciate the review